src/validation.cpp

```diff
@@ -1851,6 +1851,22 @@ bool ConnectBlock(const CBlock& block, CValidationState& state, CBlockIndex* pin
         flags |= SCRIPT_VERIFY_NULLDUMMY;
     }
 
+    if (IsArgSet("-bip148")) {
+        // BIP148 mandatory segwit signalling.
+        int64_t nMedianTimePast = pindex->GetMedianTimePast();
+        if ( (nMedianTimePast >= 1501545600) &&  // Tue 01 Aug 2017 00:00:00 UTC
```

Comment:
August 1st is the middle of the school holiday here in CZ. Many people are away from their computers/nodes. BIP148 will cause (maybe temporary) network split. This is the worst time that could have been chosen for such change.